specification/src/main/asciidoc/webprofile/Introduction.adoc

```diff
@@ -1,6 +1,6 @@
 == Introduction
 
-This specification defines the Jakarta EE Web
+This specification defines the Jakarta(TM) EE Web
```

Comment:
Not using the TM symbol, as in the line below?

Reply:
This is how I have always done the Trademark symbol in Asciidoc.  The (TM) gets rendered as the superscript TM.